judge/models/contest.py

```diff
@@ -86,7 +86,8 @@ class Contest(models.Model):
                                    default=False)
     view_contest_scoreboard = models.ManyToManyField(Profile, verbose_name=_('view contest scoreboard'), blank=True,
                                                      related_name='view_contest_scoreboard',
-                                                     help_text=_('These users will be able to view the scoreboard.'))
+                                                     help_text=_('These users will be able to view the scoreboard \
+                                                        , and view contest submisisons.'))
```

Comment:
Indent here is weird. Instead of `\`, use two separate quotes and align the start `'`. Also, probably move the comma to the line above. See the other help text for inspiration.